01-filedir.md

```diff
@@ -426,7 +426,7 @@ and we will see it in many other tools as we go on.
 > 3.  `2012-12-01/ 2013-01-08/ 2013-01-27/`
 > 4.  `original pnas_final pnas_sub`
 
-> ## FIXME {.challenge}
+> ## `ls` command construction {.challenge}
```

Comment:
On Sun, Feb 01, 2015 at 10:38:34AM -0800, Bernhard Konrad wrote:
> Great, please go ahead and update your branch @wking. I'll merge
> when you're done.

I've updated my branch, but I've only picked up one of your
suggestions directly.  For the other two headings where you suggested
changes, I've tried something a bit different.  Details in the commit
message for my just-pushed ba597324 (01-filedir.md: Fill in FIXME
challenge titles, 2014-12-18).